From ff9d306fca4da766202d03a033dd145d7477a761 Mon Sep 17 00:00:00 2001
From: Tobias Dubois <tobias.dubois@sony.com>
Date: Mon, 11 Jun 2018 16:54:51 +0200
Subject: [PATCH 07/32] Use View.getHeight() instead of Canvas.getHeight() for
 PageIndicatorDots

Use View.getHeight() instead of Canvas.getHeight() to get the same height
regardless of using hardware or software accelerated drawing.

Test: manual
Change-Id: I475afef017102d7968f28a7aab8116238af3d90d
---
 src/com/android/launcher3/pageindicators/PageIndicatorDots.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/launcher3/pageindicators/PageIndicatorDots.java b/src/com/android/launcher3/pageindicators/PageIndicatorDots.java
index 709975f02..f7c730a6c 100644
--- a/src/com/android/launcher3/pageindicators/PageIndicatorDots.java
+++ b/src/com/android/launcher3/pageindicators/PageIndicatorDots.java
@@ -244,7 +244,7 @@ public class PageIndicatorDots extends View implements PageIndicator {
         float startX = (getWidth() - mNumPages * circleGap + mDotRadius) / 2;
 
         float x = startX + mDotRadius;
-        float y = canvas.getHeight() / 2;
+        float y = getHeight() / 2;
 
         if (mEntryAnimationRadiusFactors != null) {
             // During entry animation, only draw the circles
-- 
2.17.1

